### PR TITLE
Upgraded to Python 3.9.1

### PR DESCRIPTION
**Pull request by webbyfox**
This changeset upgrades great-cms to python 3.8.7

- [x] Upgraded any vulnerable dependencies.
- [x] Python requirements have been re-compiled.

### CODE REVIEW

**Review by depsiatwal — COMMENTED** [review]
LGTM do we need to update readme

**Comment by webbyfox** [comment]
@depsiatwal Updated READme

**Review by depsiatwal — APPROVED** [review]
LGTM

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ install_postgresql_client: &install_postgresql_client
     name: Install PostgreSQL Client
     command: |
       sudo apt-get update
-      sudo apt-get install postgresql-client-9.6
```
Comment (stevejalim) [inline .circleci/config.yml:36]: Are we on 10? I thought @depsiatwal upgraded us after Christmas maybe?

**Review by stevejalim — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (depsiatwal) [reply .circleci/config.yml:36]: ahh yes we are may be best to use same as server version

**Pull request merged**